src/main/java/io/vertx/core/http/impl/Http2HeadersAdaptor.java

```diff
@@ -228,6 +236,24 @@ public boolean contains(CharSequence name) {
     return headers.contains(toLowerCase(name));
   }
 
+  @Override
+  public boolean contains(String name, String value, boolean caseInsensitive) {
+    if (headers instanceof DefaultHttp2Headers) {
+      DefaultHttp2Headers h = (DefaultHttp2Headers) headers;
```

Comment:
perhaps we should submit a patch for Netty `Http2Headers` and add this method ? this look a bit awkward - wdyt @normanmaurer

Reply:
What exactly you want to submit ?

Reply:
to have `Http2Headers` declare the method `contains(String name, String value, boolean caseInsensitive)` or with CharSequence

Reply:
@normanmaurer I just pushed a commit in my fork to give you an idea: https://github.com/tsegismont/netty/commit/2fb2d359586bfe54f6f03445a76a1b1527e924fc It would be great if you can take a look.

Reply:
@vietj since the `Http2HeadersAdaptor` changes are the only thing left discussed, how about reverting them and merge this PR? We can revisit if the commit above lands in Netty and, in the meantime, it will work in HTTP2 with the default impl from `Multimap`

Reply:
@normanmaurer how can we move the Netty contrib forward? Should I open an issue on your side?

Reply:
> since the Http2HeadersAdaptor changes are the only thing left discussed, how about reverting them and merge this PR?

@vietj ^^

Reply:
@tsegismont just reviewed your patch... please open a PR and sorry for the delay.